ssg/build_remediations.py

```diff
@@ -747,15 +689,15 @@ def expand_xccdf_subs(fix, remediation_type, remediation_functions):
             <sub idref="function_function_name"/>
             function_name "arg1" "arg2" ... "argN"
     """
-
+    if fix is not None:
+        fix_text = fix.text
```

Comment:
And if it's None it would probably make sense to terminate, otherwise the variable fix_text would be undefined.